scripts/addon/build.py

```diff
@@ -0,0 +1,361 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import json
+import os
+from pathlib import Path
+import xml.etree.ElementTree as ET
+import tempfile
+import shutil
+import sys
+
+comment_types = {
+    "text": f"Standard text in a guide block",
+    "title": f"Title in a guide block",
+    "ulist": f"Bullet unordered list item in a guide block",
+    "olist": f"Bullet ordered list item in a guide block",
+}
+
+
+def retrieve_strings_tutorial(manifest, filename):
+    tutorial_strings = {}
+
+    tutorial_json = manifest["tutorial"]
+    if not "id" in tutorial_json:
```

Comment:
```suggestion
    if "id" not in tutorial_json:
```

nit: [Pythonic way](https://www.flake8rules.com/rules/E713.html)